src/screens/help.ts

```diff
@@ -132,12 +133,19 @@ export default async function helpScreen(): Promise<string> {
 			}
 
 			if (details.accepts) {
-				fullDescription += chalk.gray.italic(` (accepts: ${details.accepts.join(', ')})`);
+				const accepts =
+					details.accepts instanceof Array
+						? details.accepts
+						: isPromiseCallback(details.accepts)
+						? await details.accepts()
+						: details.accepts();
```

Comment:
Instead of needing a type guard, why not just do something like this?

```suggestion
				const arrayOrPromise = typeof details.accepts === 'function' ? details.accepts() : details.accepts;
				const accepts = arrayOrPromise instanceof Promise ? await arrayOrPromise : arrayOrPromise;
```

Reply:
hmmmmm.  call the function regardless, and see if the result is a Promise? ok.

Reply:
Right; that's what I was trying to explain earlier.